db/error_handler_test.cc

```diff
@@ -237,7 +239,6 @@ TEST_F(DBErrorHandlingTest, CorruptionError) {
   Destroy(options);
 }
 
-#ifndef TRAVIS
```

Comment:
is it fixed in travis?

Reply:
Yes! I didn't have a way of reproducing the travis failure before, but luckily the regression tests caught it.